openedx/core/djangoapps/programs/utils.py

```diff
@@ -334,31 +330,99 @@ def _attach_course_run_upgrade_url(self, run_mode):
         else:
             run_mode['upgrade_url'] = None
 
-    def _attach_instructors(self):
-        """
-        Extend the program data with instructor data. The instructor data added here is persisted
-        on each course in modulestore and can be edited in Studio. Once the course metadata publisher tool
-        supports the authoring of course instructor data, we will be able to migrate course
-        instructor data into the catalog, retrieve it via the catalog API, and remove this code.
-        """
+    def _attach_course_run_can_enroll(self, run_mode):
```

Comment:
I suppose we can move it. I wonder why we don't need this check on the "my programs" page.